src/api/api.cpp

```diff
@@ -284,6 +292,14 @@ struct custom_operation
     {
         return op.compute(std::move(ctx), std::move(output_shape), std::move(inputs));
     }
+
+    std::ptrdiff_t output_alias(std::vector<shape> inputs) const
+    {
+        // TODO: For now, only support one output alias
+        return op.output_alias(std::move(inputs)).front();
```

Comment:
We should check the size of the vector. If its empty we should return `-1`, and if its greater then `1` we should throw.